docs/calicoctl/node.md

```diff
@@ -0,0 +1,211 @@
+                         (default value depends on calicoctl binary version)
+    <AS_NUM>: Autonomous System number to use for BGP peering.
+              (default global AS number of 64511 is used if not specified)
+    <LOG_DIR>: Directory where Calico will store logs, if not default.
+               (default: /var/log/calico)
+    <DETACH>: Boolean to have calico/node run as detached (true) or in the foreground (false).
+              (default: true)
+    <LIBNETWORK_IMAGE_NAME>: Desired calico/node-libnetwork Docker image to use when 
+                             using the Docker libnetwork driver.
+
+    --kubernetes: Download and install the kubernetes plugin.
+    --rkt: Download and install the rkt plugin.
+    --libnetwork: Download and run the calico/node-libnetwork Docker image.
+```
+
+When running the `calicoctl node` command with the `--libnetwork` plugin, the 
+command starts a container using the `calico/node-libnetwork` Docker image in 
+addition to starting the `calico/node` Docker image.
+
+Examples:
+
+```
+$ calicoctl node
+
+No IP provided. Using detected IP: 172.25.0.1
+Calico node is running with id: c95fc492d57bd7d3c568e5b1d67001c1cec7c01b771531618fbf910557e37f29
+
+# Run the Calico node with IPv4 and IPv6, using calico/node version v0.7.0
+$ calicoctl node --ip=172.25.0.1 --ip6=2620:0104::1 --node-image=calico/node:v0.7.0
+Pulling Docker image calico/node:v0.7.0
+
+Calico node is running with id: f97a6fe29109ea6d9cc3be70a2a6fd9b56a5dc3c4e9ba77f6b14643ec3da4915
+
+# Run the Calico node using the Docker libnetwork driver 
+$ sudo calicoctl node --libnetwork
+No IP provided. Using detected IP: 172.25.0.1
+Calico node is running with id: c95fc492d57bd7d3c568e5b1d67001c1cec7c01b771531618fbf910557e37f29
+Calico libnetwork driver is running with id: 504b1d6d42908e376d9941ad8e3dfd65b072c15455c108e0205beee52d71fa69
+```
+
+### calicoctl node stop 
+This command is used to stop a `calico/node` instance.  If there exist 
+endpoints that have been networked with Calico, a warning message will appear.
+
+This command must be run as root and must be run on the specific Calico node 
+that you are configuring.
+
+Command syntax:
+
+```
+calicoctl node stop [--force]
+
+    --force: Stop the node instance, regardless of warnings.
+```
+
+Examples:
+
+```
+$ calicoctl node stop
+Node stopped and all configuration removed
+```
+
+### calicoctl node bgp peer add <PEER_IP> as <AS_NUM>
+This command allows users to configure specific BGP peers with this node.
+
+This command must be run on the specific Calico node that you are configuring.
+
+Command syntax:
+
+```
+calicoctl node bgp peer add <PEER_IP> as <AS_NUM>
+
+    <PEER_IP>: IP address of BGP peer to add.
+    <AS_NUM>: Autonomous systems number to configure with BGP peer.
+
+```
+
+Examples:
+
+```
+$ calicoctl node bgp peer add 172.25.0.2 as 65511
+```
+
+### calicoctl node bgp peer remove <PEER_IP>
+This command allows users to remove specific BGP peers from this Calico node.
```

Comment:
"NOTE: This command only removes peers configured with `calicoctl node bgp peer add`.  It does not remove global peers (`calicoctl bgp peer add`) or peerings with other Calico nodes if the node mesh (`calicoctl bgp node-mesh`) is on."